### PR TITLE
Update RV2 to use Clang 17

### PR DESCRIPTION
**Pull request by JonatanAntoni**

### CODE REVIEW

File: Layer/Target/CM33_FP/RTE/Device/IOTKit_CM33_FP_VHT/RTE_Device.h
Comment (VladimirUmek) [inline Layer/Target/CM33_FP/RTE/Device/IOTKit_CM33_FP_VHT/RTE_Device.h:1]: Renaming CM3_VHT -> CM33_FP?

**Review by VladimirUmek — CHANGES_REQUESTED** [review]
Please also update REDME.md

CI is not working - do we wait till next toolbox release or do we implement workflow workaround?

File: Layer/Target/CM3/RTE/Device/CMSDK_CM3_VHT/RTE_Device.h
[the remark this answers is not in the document]
Comment (JonatanAntoni) [reply Layer/Target/CM3/RTE/Device/CMSDK_CM3_VHT/RTE_Device.h:1]: This is only git artifact.

File: Project/.clangd
Comment (VladimirUmek) [inline Project/.clangd:1]: This file probably shouldn't be here?

Comment (JonatanAntoni) [reply Project/.clangd:1]: True, good catch!

File: Project/README.md [hunk 1]
```diff
@@ -8,92 +8,112 @@ Independent project can be generated for specific RTOS, DEVICE and COMPILER. Pos
...
-$ cbuild Validation.csolution.yml --update-rte --context .{RTOS}+{DEVICE} --toolchain {COMPILER}
-```
+ ./Project $ cbuild Validation.csolution.yml --update-rte --context .{RTOS}+{DEVICE} --toolchain {COMPILER}
+ ```
+
 Executable file is then used as input parameter when calling Virtual Hardware Target model to run the validation:
+
 ```Shell
-$ {VHT_MODEL} -f ../Layer/Target/{DEVICE}_VHT/vht_config.txt -a Validation.{RTOS}+{DEVICE}_{COMPILER}/Validation.{RTOS}+{DEVICE}_OutDir/Validation.axf
+ ./Project $ {VHT_MODEL} -f ../Layer/Target/{DEVICE}/model_-config.txt -a Validation.{RTOS}+{DEVICE}_{COMPILER}/Validation.{RTOS}+{DEVICE}_OutDir/Validation.axf
```
Comment (VladimirUmek) [inline Project/README.md:99]: {VHT_MODEL} should be {AVH_MODEL} as per table below

Comment (JonatanAntoni) [reply Project/README.md:99]:
```suggestion
 ./Project $ {AVH_MODEL} -f ../Layer/Target/{DEVICE}/model_-config.txt -a Validation.{RTOS}+{DEVICE}_{COMPILER}/Validation.{RTOS}+{DEVICE}_OutDir/Validation.axf
```

**Review by VladimirUmek — APPROVED** [review]
Looks good! Thanks again!

**Pull request merged**